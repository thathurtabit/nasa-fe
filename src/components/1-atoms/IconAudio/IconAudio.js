import React from 'react';
import PropTypes from 'prop-types';
import SVG from './IconAudio.styled';

const IconAudio = props => (
  <SVG width={50} height={50} viewBox="0 0 25 25" {...props}>
    <title>{props.title}</title>
    <path d="M5 17h-5v-10h5v10zm2-10v10l9 5v-20l-9 5zm17 4h-5v2h5v-2zm-1.584-6.232l-4.332 2.5 1 1.732 4.332-2.5-1-1.732zm1 12.732l-4.332-2.5-1 1.732 4.332 2.5 1-1.732z" />
  </SVG>
);

export default IconAudio;

IconAudio.propTypes = {
  title: PropTypes.string.isRequired,
};
