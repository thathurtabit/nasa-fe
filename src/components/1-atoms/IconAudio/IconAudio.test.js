import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import 'jest-styled-components';
import { shallowWithTheme } from '../../../utils/test_config/testHelpers';

import IconAudio from './IconAudio';

const title = 'test title';

describe('IconAudio', () => {
  it('should render correctly', () => {
    const element = shallowWithTheme(<IconAudio title={title} />);
    expect(shallowToJson(element)).toMatchSnapshot();
  });
});
