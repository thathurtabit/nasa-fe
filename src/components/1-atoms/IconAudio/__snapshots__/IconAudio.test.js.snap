// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconAudio should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
